community_images/common/orchestrator/main.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, args):
         self.docker_client = docker.from_env()
         self.publish = args.publish
         self.force_publish = args.force_publish
-        self.config_name = self.args.config
```

Comment:
Why is this required

Reply:
if you run the command

python3 common/orchestrator/main.py hourly_run traefik/traefik/

 from 
/rapidfort/community-images/community_images

then extra "/" at the end of traefik/traefik/ makes out the calculation of directory 3 level above instead of two , which results in file not found. the extra "/" is automatically added because directory selection was made by pressing tab.